src/test/trig.py

```diff
@@ -1,28 +0,0 @@
-"""
```

Comment:
Could we leave the directory structure as it was for this PR? Prefer keeping refactor PRs like that separate from feature PRs like adding arcsin, makes things easier in the future when looking back at this PR to see what changed